frontend/src/router/layouts/MainLayout.scss

```diff
@@ -0,0 +1,27 @@
+.layout.main {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+
+  & > .container {
+    flex: 1;
+  }
+
+  & > footer {
+    min-height: 4.1rem;
+
+    .quick-links {
+      gap: 0 1.5em;
+
+      a {
+        color: var(--bs-white);
+        text-decoration: none;
```

Comment:
Just curious to know what made you to use `var()` instead of variables in SCSS.

Reply:
I'm trying to un-learn the habit of using SCSS vars for things like this when CSS variables are available. At the end of the day in our site it's going to be the same, but CSS variables let us change things at runtime: If the user switches to a dark theme, or if the client wants us to fade out the colors when we mouse over or something, we would just use JS to change a couple of variables, while we might need to override a lot more if the values were "hardcoded" from compiled SCSS.

Reply:
...but I haven't used Bootstrap in a long time and I'm pretty new to styling React components, so let me know if there's a better way to do things! CSS Variables didn't exist last time I used Bootstrap 😆